eox_core/cms/static/js/CourseRerun.jsx

```diff
@@ -0,0 +1,375 @@
+
+    this.state = {
+      courseKey: '',
+      organizationList: '',
+      organizationListTextArea: '',
+      initialOrganizationList: [],
+      openAlert: false,
+      completedTasks: [],
+      failedTasks: [],
+      statusAlertMessage: '',
+      statusAlertType: '',
+      isLoading: false,
+      courseRun: '',
+      courseName: '',
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.onCloseAlert = this.onCloseAlert.bind(this);
+    this.handleOrganizationListChange = this.handleOrganizationListChange.bind(this);
+  }
+
+  componentDidMount() {
+    this.getOrganizationList();
+  }
+
+  handleChange(value, name) {
+    if (value !== name) {
+      this.setState({
+        [name]: value,
+      });
+    }
+  }
+
+  getOrganizationList() {
+    this.setState({
+      isLoading: true,
+    });
+
+    clientRequest(
+      this.organizationApiUrl,
+      'GET',
+    ).then(
+      res => this.handleResponse(res),
+    ).then((response) => {
+      this.fillOrganizationList(response);
+    }).catch((error) => {
+      this.openStatusAlert(`An error occurred while getting the organizations list: ${error.message}`, 'danger');
+      this.setState({
+        isLoading: false,
+      });
+    });
+  }
+
+  handleSubmit() {
+    const isValid = this.onSubmitValidator();
+    /* eslint-disable no-undef */
+    if (isValid && confirm(`Rerun ${this.state.courseKey} course into ${this.state.organizationList.length} organizations.`)) {
+      this.setState({
+        completedTasks: [],
+        failedTasks: [],
+      });
+      this.courseKeyExists().then(
+        (courseExists) => {
+          this.setState({
+            isLoading: false,
+          });
+          if (courseExists && this.organizationListExists()) {
+            this.submitRerunCourse();
+          }
+        },
+      );
+    }
+  }
+
+  submitRerunCourse() {
+    let requestTimeOut = 0;
+    const courseKeyData = this.getCourseKeyData();
+
+    for (const organization of this.state.organizationList) {
+      setTimeout(() => {
+        this.setState({
+          isLoading: true,
+        });
+        const requetsBody = {
+          source_course_key: this.state.courseKey,
+          org: organization,
+          number: courseKeyData.number,
+          display_name: this.state.courseName,
+          run: this.state.courseRun,
+        };
+
+        clientRequest(
+          this.courseApiUrl,
+          'POST',
+          requetsBody,
+        ).then(
+          res => this.handleCourseRerunResponse(res, organization),
+        ).catch((error) => {
+          this.openStatusAlert(error.message, 'danger');
+          this.setState({
+            isLoading: false,
+          });
+        });
+      }, requestTimeOut);
+      requestTimeOut += this.props.requestTimeOut;
+    }
+  }
+
+  handleCourseRerunResponse(response, organization) {
+    const actualSuccessTasks = this.state.completedTasks;
```

Comment:
I think `const actualSucceededTasks` could be a better name

Reply:
@diegomillan Addressed.